main.go

```diff
@@ -46,6 +50,19 @@ func init() {
 func validateCompliance(_ *cobra.Command, args []string) {
 	setLogLevel(logLevel)
 
+	for module, licence := range overriddenModuleLicences {
+		cmd := exec.Command("go", "list", "-m", "-f", "\"{{.Dir}}\"", module)
+		cmd.Env = os.Environ()
+		cmd.Env = append(cmd.Env, "GO111MODULE=on")
+
+		out, err := cmd.CombinedOutput()
```

Comment:
Think this should be `.Output()` just in case `go list` prints unrelated stuff to stderr.